www/_posts/2016-03-16-new-plugin-fetching.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+author:
+    name: Richard Knoll
+    url: https://github.com/riknoll
+title:  "Upcoming Changes to Plugin Fetching"
+categories: announcements
+tags: news
+---
+
+The Cordova 6.0.0 release introduced the pinning of core plugin
+versions in cordova-lib.
+
+We are happy to announce that one of the new features in the upcoming
+Cordova 6.1.0 release is a general API that allows any plugin to guide
+the CLI in choosing a compatible plugin release to fetch for a given project.
+This moves the plugin dependency information out of cordova-lib so that it
+can update independently of the Cordova tools and support third-party plugins
+outside of core. Our hope is that this feature will improve Cordova's plugin
+ecosystem and reduce some of the frustration that Cordova developers face when
+adding a new plugin to a project.
+
+<!--more-->
+
+The new API allows for plugin developers to give a mapping of plugin
+releases to project requirements in their `package.json`. When an app developer
+adds a plugin to their Cordova project, cordova-lib will examine their
+installed plugins, platforms, and cordova-lib version and decide which
+version of the plugin to fetch based on the provided mapping.
+If the project does not fulfill the requirements for the latest release of the
+plugin, the CLI will print warnings indicating the problems and fetch the
+latest release of the plugin that is compatible.
+
+This API may eventually replace the [engines element](http://cordova.apache.org/docs/en/latest/plugin_ref/spec.html#engines-and-engine) in `plugin.xml`.

```

Comment:
I would say it **will** eventually replace the engine element in plugin.xml! But we can wait until we actually make that change